File: MiwokApp/app/src/main/java/com/example/miwokapp/MainActivity.java
package com.example.miwokapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.TextView;

import com.google.android.material.tabs.TabLayout;

import org.w3c.dom.Text;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ViewPager viewPager = (ViewPager) findViewById(R.id.viewpager);

        // Create an adapter that knows which fragment should be shown on each page
        SimpleFragmentPagerAdapter adapter = new SimpleFragmentPagerAdapter(getSupportFragmentManager());

        // Set the adapter onto the view pager
        viewPager.setAdapter(adapter);

        TabLayout tabLayout = findViewById(R.id.slidingtabs);
                tabLayout.setupWithViewPager(viewPager);

    }
}

File: UberClone/app/src/main/java/com/example/uberclone/MainActivity.kt
package com.example.uberclone

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.Switch
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.switchmaterial.SwitchMaterial

class MainActivity : AppCompatActivity() {
    var getStartedButton: Button? = null
    var switch: SwitchMaterial? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        getStartedButton = findViewById(R.id.getStartedButton)
        switch = findViewById(R.id.switch1)
        getStartedButton!!.setOnClickListener {
            val intent: Intent
            if (switch!!.isChecked) {
                intent = Intent(this, DriverActivity::class.java)
                startActivity(intent)
            } else {
                intent = Intent(this, RiderActivity::class.java)
                startActivity(intent)
            }
        }
        supportActionBar?.hide()
    }
}